controllers/kustomization_controller.go

```diff
@@ -592,6 +636,10 @@ func (r *KustomizationReconciler) applyWithRetry(kustomization kustomizev1.Kusto
 }
 
 func (r *KustomizationReconciler) prune(kustomization kustomizev1.Kustomization, snapshot *kustomizev1.Snapshot, force bool) error {
+	if kustomization.Spec.KubeConfig != nil {
+		// TODO: implement pruning for remote clusters
```

Comment:
Maybe (debug) log something here that it is currently not supported and will be skipped?